docs/spec_draft.md

```diff
@@ -345,6 +346,66 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.bitcast_convert
+
+### Semantics
+
+Performs an element-wise bitcast operation on `operand` tensor and produces a
+`result` tensor where the bits are reinterpreted with the new element type.
+Conversions between different bitwidths are not element-wise and creates/deletes
+a dimension if new element type requires less/more bits, respectively.
+
+Bitcast is implemented as a low-level cast, so machines with different
+floating-point representations (e.g. endianesss) will give different results.
+
+Let `E` and `E'` be the `operand` and `result` element type respectively, and
+`R = rank(operand)`:
+  * If `numBits(E) == numBits(E')`,
+     `bitRepr(result[i0, ..., iR-1]) = bitRepr(operand[i0, ..., iR-1])`.
+  * If `numBits(E) > numBits(E')`,
+    `bitRepr(result[i0, ..., iR-1, :]) = bitRepr(operand[i0, ..., iR-1])`.
+  * If `numBits(E') > numBits(E)`,
```

Comment:
`numBits(E) < numBits(E')`
reverse the order to keep in consistent with conditions above